seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -853,7 +964,18 @@
         "assert(myStr === \"I am a \\\"double quoted\\\" string inside \\\"double quotes\\\"\", 'message: Variable myStr should equal to (<code>\"I am a \"double quoted\" string inside \"double quotes\"\"</code>).');"
       ],
       "type": "waypoint",
-      "challengeType": 1
+      "challengeType": 1,
+      "nameEs": "Escapando Citas Literales en Cadenas",
```

Comment:
quote también se traduce como entrecomillar ( http://translation.babylon.com/spanish/to-english/entrecomillar/ ).  En mi humilde opinión a un lector le va a quedar más facil esta lección si traduces quote como "comillas".  Con esto el título sería "Escapar comillas en cadenas de texto"